src/bun.js/api/server.zig

```diff
@@ -190,6 +190,7 @@ pub const ServerConfig = struct {
     }
 
     pub const SSLConfig = struct {
+        requires_custom_request_ctx: bool = false,
```

Comment:
I'm not convinced this boolean is necessary

Reply:
its because there are certain cases where the object is passed but we do not need a custom context